.github/workflows/scripts/countline.py

```
@@ -0,0 +1,295 @@
+
+    # Get interesting filepaths
+    repo_filepath_list = _filepaths_in_directories(directories)
+
+    # Iterate and analyze each directory
+    for filepath in repo_filepath_list:
+        # Skip excluded files
+        if filepath in exclude_list:
+            continue
+
+        # Skip /node_modules/ sub directories
+        if "{0}node_modules{0}".format(os.sep) in filepath:
```

Ruff (0.12.2)

Use f-string instead of `format` call

Convert to f-string

(UP032)